certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG

```diff
@@ -88,6 +99,7 @@ TT:
 *  before certificate maturity.
 *  Enter as comma delimited single or range values. ex. 1,10-20,30,33.
 *  VALID VALUES: 0-9999
+*  DEFAULT: Valid values must be entered - there are no default values
 TF:
 *
 *  For the following custom message settings, data will be displayed as
```

Comment:
The new CFG changes look good. I will look at the PowerOn changes tomorrow.
Please merge back in your latest change from two months ago about html tags in comments, as that is missing from this older version:

*-----------------------------------------------------------------------------------------------------------------------
* Custom Messages
*-----------------------------------------------------------------------------------------------------------------------
* For the following custom message settings, data will be displayed as written. The following HTML tags may be
* utilized to facilitate text formatting:
*   <h1>, <h2>, <h3>, <h4>, <h5>, <h6>, <b>, <strong>, <i>, <u>, <ul>, <ol>, <li>, <br>, <p> and <hr>
* Avoid the use of double quotes or special characters in your messages.
*
* For backwards compatibility, a single blank line will be interpreted as a new line and two blank lines in a row
* will be interpreted as a new paragraph
*
*Use this ruler as a guideline when setting up your messages. Do not extend past the end of this line (120 characters).
*        10        20        30        40        50        60        70        80        90        100       110       120
*23456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
*--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|

Reply:
Updated